android/anc/src/test/java/org/smartregister/fhircore/anc/ui/details/vitalsigns/VitalSignsDetailsFragmentTest.kt

```diff
@@ -177,7 +177,9 @@ internal class VitalSignsDetailsFragmentTest : RobolectricTest() {
           "23",
           "bpunit",
           "23",
-          "pulseunit"
+          "pulseunit",
+          "22.5",
+          "kg/m2"
```

Comment:
Add name to function parameters to improve readability

Reply:
done